File: LotteryGenius.API/ViewModels/MegaWinnerViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using LotteryGenius.API.Data.Entities;

namespace LotteryGenius.API.ViewModels
{
    public class MegaWinnerViewModel
    {
        public int id { get; set; }
        public string ball1 { get; set; }
        public string ball2 { get; set; }
        public string ball3 { get; set; }
        public string ball4 { get; set; }
        public string ball5 { get; set; }
        public string megaball { get; set; }
        public string megaplier { get; set; }
        public DateTime draw_date { get; set; }
        public IEnumerable<MegamillionWinners> picks { get; set; }
    }
}

File: LotteryGenius.Web/src/app/viewmodels/megawinnerviewmodel.ts
import { MegaWinners } from '../models/megawinners';

export class MegaWinnerViewModel {
    id: number;
    ball1: string;
    ball2: string;
    ball3: string;
    ball4: string;
    ball5: string;
    megaball: string;
    megaplier: string;
    draw_date: Date;
    jackpot: string;
    picks: Array<MegaWinners>;
}